docs/source/installation_instructions.rst

```diff
@@ -74,18 +50,31 @@ start menu and running the following line.
 If the Miniconda installation was successful, a list of available packages will appear.
 
 
-Pytentiostat Package Installation
+pytentiostat Package Installation
 ----------------------------------
 
-Once python and the conda are installed, the pytentiostat package installation is simple. Follow these steps to complete
-the Pytentiostat installation!
+Once the miniconda installation is complete, proceed with the pytentiostat package installation as follows.
 
 1. Open the anaconda prompt from the start menu.
-2. Navigate to the directory you would like to install the software, e.g. desktop
-3. In the command line, type:
+2. Create a conda environment where *mycondaenv* is the name you assign to your environment and *x.x* is the version of Python
```

Comment:
I think it might be better to just specify ``python=3`` and not tie it to any particular minor version.  It is a bit simpler.